src/viur/core/skeleton/instance.py

```diff
@@ -324,6 +324,11 @@ def structure(self) -> dict:
             for i, (key, bone) in enumerate(self.items())
         }
 
+    def values(self):
+        return {
+            bone_name: bone.value(self, bone_name) for bone_name, bone in self.items()
+        }
+
```

Comment:
Now we have the `values` methode here and here https://github.com/viur-framework/viur-core/blob/c1e9d700c2cc4d7f78e3a39b2dc6afb546802c2e/src/viur/core/skeleton/instance.py#L115

Reply:
@phorward

Reply:
You're right... I didn't recognize this.

What do you think of using the wording *simplified* for this case, rather than *get_values*? `skel.simplified()` is like `skel.values()`, but returns the plain, simplified version of the values, which can be simply serialized to JSON, BSON, XML... ?